doc/tools/west/without-west.rst

```diff
@@ -0,0 +1,46 @@
+.. _no-west:
+
+Using Zephyr without west
+#########################
+
+There are many valid usecases to avoid using a meta-tool such as west which has
+been custom-designed for a particular project and its requirements.
+This section applies to Zephyr users who fall into one or more of these
+categories:
+
+- Already use a multi-repository management system (for example Google repo)
+
+- Do not need additional functionality beyond compiling and linking
+
+In order to obtain the Zephyr source code and build it without the help of
+west you will need to manually clone the repositories listed in the
+`manifest file`_ one by one, at the path indicated in it.
+
+.. code-block:: console
+
+   mkdir zephyrproject
+   cd zephyrproject
+   git clone https://github.com/zephyrproject-rtos/zephyr
+   # clone additional repositories listed in the manifest file
+
+If you want to manage Zephyr's repositories without west but still need to
+use west's additional functionality (flashing, debugging, etc.) it is possible
+to do so by manually creating an installation:
+
+.. code-block:: console
+
+   # cd into zephyrproject if not already there
+   git clone https://github.com/zephyrproject-rtos/west.git
```

Comment:
This is not consistent with your comment here: https://github.com/zephyrproject-rtos/zephyr/issues/12812#issuecomment-458750345

Does it need to be cloned in `.west/west`?